app/src/pages/admin/allUsersPage/allUsersGrid/projectsAndRolesColumn/projectsAndRolesColumn.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './projectsAndRolesColumn.scss';
+
+const cx = classNames.bind(styles);
+
+export class ProjectsAndRolesColumn extends Component {
+  static propTypes = {
+    className: PropTypes.string.isRequired,
+    value: PropTypes.object,
+  };
+  static defaultProps = {
+    value: {},
+  };
+  state = {
+    expanded: false,
+  };
+  handleExpand = () => {
+    this.setState({ expanded: !this.state.expanded });
+  };
+  render() {
+    const { className, value } = this.props;
+    const allProjects = Object.keys(value.assignedProjects);
+    const firstProjects = allProjects.slice(0, 2);
+    const restProjects = allProjects.slice(2);
+    return (
+      <div className={cx('projects-and-roles-col', className)}>
+        <span className={cx('mobile-label', 'projects-and-roles-mobile-label')}>
+          Projects and roles
+        </span>
+        {firstProjects.map((project, index) => (
+          <span key={project} className={cx('project')}>
+            {project}
+            {index < firstProjects.length - 1 && ', '}
+            <span className={cx('hidden', 'shown-on-mobile')}>
+              {index >= 0 && ', '}
+              <span className={cx('project-role')}>
+                {value.assignedProjects[project].projectRole.toLowerCase().replace(/_/g, ' ')}
```

Comment:
What if a project has `_` symbol in the name? Is it necessary to apply such formatting?

Reply:
This formatting is using for Project Role only, because api response looks like "PROJECT_MANAGER" f.e. Should I use constants and switch statement instead?

Reply:
No, it's ok. Just move this to a function `formatProjectRole`